src/postgres/contrib/yb_auh/yb_auh.c

```diff
@@ -205,7 +205,7 @@ static void tserver_collect_samples(TimestampTz auh_sample_time)
   for (int i = 0; i < numrpcs; i++) {
     auh_entry_store(auh_sample_time, rpcs[i].metadata.top_level_request_id,
                     rpcs[i].metadata.current_request_id, rpcs[i].wait_status_code,
-                    rpcs[i].aux_info.tablet_id, rpcs[i].metadata.top_level_node_id,
+                    rpcs[i].aux_info.method, rpcs[i].metadata.top_level_node_id,
```

Comment:
I think it may be more useful to have tablet-id for slicing/dicing purposes than the method name, while exporting to the circular buffer/archival storage.

But feel free to use the method name for now, while we are debugging/understanding auh